packages/peregrine/lib/talons/ProductFullDetail/useProductFullDetail.js

```diff
@@ -224,27 +222,27 @@ export const useProductFullDetail = props => {
                 addItemMutation = addConfigurableProductToCart;
             }
 
-            addItemToCart({
+            await addItemToCart({
                 ...payload,
                 addItemMutation,
-                fetchCartDetails,
                 fetchCartId
             });
+            toggleDrawer('cart');
```

Comment:
Importantly, we missed this in #2017 and we were calling `toggleDrawer` when we updated a configurable item, after the add.